packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -71,4 +89,30 @@ export default class UpdateConfigure extends EasCommand {
       })}`
     );
   }
+
+  private sanitizeFlags(flags: RawUpdateConfigureFlags): UpdateConfigureFlags {
+    if (
```

Comment:
would be good to figure out a way to factor out this code since we'll be adding it to so many commands, maybe into the validator of the enum flag definition?